### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by IrinaBreiksh**
Check homework 2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+<h1>Моя первая страница</h1>
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Title</title>
-</head>
-<body>
+  </head>
```
Comment (5tronciy) [inline index.html:7]: Open and close tags should have the same padding.

Comment (5tronciy) [reply index.html:7]: Hereinafter use prettier before commit.

File: index.html [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+        Именно эти языки отвечают за то, как сайт будет выглядеть в браузере. Выучить их можно <del>самостоятельно с помощью интернет-ресурсов</del>. <mark>Главное</mark> - сразу применять на практике, создавая подобия сайтов. Самое главное на этом этапе - не выучить все наизусть, а получить представление о языках, их работе, атрибутах и возможностях. Когда вы будете понимать, что вам конкретно нужно, вы сможете быстро загуглить нужный тег, важнее понимать структуру в целом.
+        <h3><em>Верстка макетов</em></h3>
+        Вы можете найти гайды по верстке в интернете и шаг за шагом учиться превращать PSD-макеты в веб-страницы.
+        На этом этапе вы уже можете пробоваться на должность HTML-верстальщик.
+        <h3><em>JavaScript</em></h3>
+        Это язык программирования, который позволяет вам создать динамически обновляемый контент, управляет мультимедиа, анимирует изображения.
+
+      <h2>Советы:</h2>
+    <p>
+        1 Копайте глубже <br>
```
Comment (5tronciy) [inline index.html:26]: Better use [ordered list](https://developer.mozilla.org/ru/docs/Web/HTML/Element/ol) here.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
The main part of the task must be done in the `about/index.html` - it is harder to check it in different files.

You didn't use tags `<sub>` & `<sup>`.

Other remarks in comments.

File: index.html (hunk 2 above)
Comment (IrinaBreiksh) [reply index.html:26]: Ok, but such type of elements we were explained in Lesson 3. Should I use it here?

File: index.html (hunk 1 above)
Comment (IrinaBreiksh) [reply index.html:7]: Ok

**Comment by IrinaBreiksh** [comment]
> The main part of the task must be done in the `about/index.html` - it is harder to check it in different files.
> 
> You didn't use tags `<sub>` & `<sup>`.
> 
> Other remarks in comments.

Ok, changed

**Review by 5tronciy — APPROVED** [review]
Good work.

File: index.html (hunk 2 above)
Comment (5tronciy) [reply index.html:26]: On your choice

Comment (IrinaBreiksh) [reply index.html:26]: I'll do it in next task

**Pull request merged**